src/coreclr/src/pal/src/map/map.cpp

```diff
@@ -2413,15 +2424,28 @@ void * MAPMapPEFile(HANDLE hFile, off_t offset)
     }
 #endif // _DEBUG
 
+    size_t headerSize;
+    headerSize = GetVirtualPageSize(); // if there are lots of sections, this could be wrong
+
+    if (forceOveralign)
+    {
+        loadedBase = ALIGN_UP(loadedBase, ntHeader.OptionalHeader.SectionAlignment);
+        headerSize = ntHeader.OptionalHeader.SectionAlignment;
+        char *mapAsShared = EnvironGetenv("PAL_MAP_READONLY_PE_HUGE_PE_AS_SHARED");
```

Comment:
I am trying to parse the env var name. Does the HUGE_PE part mean HUGE_PAGE? Maybe we could make it 
```suggestion
        char *mapAsShared = EnvironGetenv("PAL_MAP_READONLY_PE_HUGE_PG_AS_SHARED");
```

Reply:
Seems I had PE PE PE fever on the brain when I wrote all of this. @MichalStrehovsky caught the last one of those and this isn't any better. I think you're right, I meant HUGE_PAGE here.